### ISSUE TITLE
Process out of memory

### ISSUE THREAD

**Issue by Art2B** [shoreditch-ops/minigun#30]
Hi ! 

I've been using Minigun to send requests with base64 img in request params. But i've got the following error when I'm launching minigun:

```
[minigum_process_out_of_memory.zip](https://github.com/shoreditch-ops/minigun/files/66994/minigum_process_out_of_memory.zip)

Phase undefined (Test POST Requests) started - duration: 2000s

<--- Last few GCs --->

    8991 ms: Scavenge 1399.0 (1454.4) -> 1399.0 (1454.4) MB, 1.8 / 0 ms [allocation failure].
    8993 ms: Scavenge 1399.0 (1454.4) -> 1399.0 (1454.4) MB, 1.8 / 0 ms [allocation failure].
    8995 ms: Scavenge 1399.0 (1454.4) -> 1399.0 (1454.4) MB, 1.8 / 0 ms [allocation failure].
    9958 ms: Mark-sweep 1399.0 (1454.4) -> 1399.0 (1454.4) MB, 963.7 / 0 ms [last resort gc].
   10906 ms: Mark-sweep 1399.0 (1454.4) -> 1399.0 (1454.4) MB, 947.5 / 0 ms [last resort gc].


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x1ba9c0137339 <JS Object>
    2: squashTokens [/home/arthur/.nvm/versions/node/v4.1.1/lib/node_modules/minigun/node_modules/minigun-core/node_modules/mustache/mustache.js:234] [pc=0x36eeedddf82] (this=0x1ba9c01b63c1 <JS Global Object>,tokens=0x23bc9027b409 <JS Array[15085174]>)
    3: parseTemplate(aka parseTemplate) [/home/arthur/.nvm/versions/node/v4.1.1/lib/node_modules/minigun/node_modules/minigun-core/node_modules/...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
Abandon (core dumped)
```

I've joined a zip file with the minigun params i use.

**Comment by hassy** [comment]
Thanks for the detailed bug report - I can reproduce the error and will investigate :+1:

**Comment by hassy** [comment]
I think I have a fix but will need to test it further before merging and publishing a new version.

**Comment by hassy** [comment]
@Art2B Published `v1.2.15` which should fix the issue.
